playbooks/reboot-hosts.yml

```diff
@@ -4,8 +4,14 @@
   gather_facts: no
   tasks:
 
+  - name: check whether consul is installed or not
+    stat:
+      path: /bin/consul
+    register: is_consul
+
```

Comment:
This task doesn't work at all, in order to get it working I suggest to add step like

failed_when: is_consul.stat.islnk is not defined